meal-planning/src/components/recipe-library/DefaultRecipes.js

```diff
@@ -0,0 +1,22 @@
+const defaultRecipes = [
+    {
+      id: 1,
+      name: 'Curry',
+      description: 'spicy coconut curry',
+      ingredients: ['ingredient']
```

Comment:
While we haven't discussed ingredients in-depth, for the sake of console errors (key error in this case), let's change the dummy data so the ingredients arrays each hold an object for every ingredient. Something like this:

```suggestion
      ingredients: [{ id: 1, name: 'ingredient' }]
```
Keys only have to be unique among **siblings**, so this should work.